async-nats/src/jetstream/context.rs

```diff
@@ -168,6 +222,29 @@ impl Context {
         }
     }
 
+    /// Convinience method that gets [Stream] if it's found on the server or creates it with
+    /// provided [Config] if it doesn't exist.
+    ///
+    /// Warning: This function does not check if returned [Config] is the same as the one passed in
```

Comment:
Probably want to fully qualify this config type?

```suggestion
    /// Warning: This function does not check if returned [Config] is the same as the one passed in
```

Reply:
it can. I'm running `cargo doc --open` and checking everything :)